components/brave_private_new_tab_ui/resources/page/components/disclaimer-dialog/index.tsx

```diff
@@ -0,0 +1,133 @@
+import * as React from 'react'
+import styled from 'styled-components'
+
+const Box = styled.div`
+  color: white;
+  font-size: 12px;
+  line-height: 18px;
+  background: rgba(12, 4, 30, 0.2);
+  max-width: 285px;
+  border-radius: 8px 0px 0px 8px;
+  display: inline-block;
+  position: relative;
+
+  p {
+    padding: 0;
+    margin: 0 0 24px 0;
+
+    &:last-child {
+      margin-bottom: 0;
+    }
+  }
+`
+
+const ScrollerBox = styled.div`
+  max-height: 300px;
```

Comment:
At the moment, the most important part of the Tor message ("[...]] you should be using Tor browser instead.") is below the fold.

Can we make it so that it shows up in full when the screen is large enough?

Reply:
@nullhook perhaps make the max-height something like `(100vh - margin - approximate search box height) / 2`?

Reply:
added this in css so the browser can do re-computes. Updated screenshot as well @fmarier - On screen size with space you won't see scroll anymore.